cmd/payout.go

```diff
@@ -0,0 +1,73 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"github.com/NodeFactoryIo/vedran/internal/script"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"strconv"
+	"strings"
+)
+
+var (
+	secret string
+	totalReward string
+	loadbalancerUrl string
+
+	totalRewardAsFloat64 float64
+)
+
+var payoutCmd = &cobra.Command{
+	Use: "payout",
+	Short: "Starts payout script",
+	Run: payoutCommand,
+	Args: func(cmd *cobra.Command, args []string) error {
+		result, err := strconv.ParseFloat(totalReward, 64)
+		if err != nil {
+			return errors.New("invalid total reward value")
+		}
+		totalRewardAsFloat64 = result
+
+		if !(strings.HasPrefix(loadbalancerUrl, "http://") ||
```

Comment:
Does it make sense to parse loadbalancerURL to url.URL?
It will check url and also it will be easier to join urls when calling endpoint.